TestBackupResumeIntegration can randomly fail

```
panic: Fail in goroutine after TestBackupResumeIntegration/resume_after_stop has completed

goroutine 3543 [running]:
testing.(*common).Fail(0xc00089a340)
	/opt/hostedtoolcache/go/1.19.12/x64/src/testing/testing.go:824 +0xe5
testing.(*common).Error(0xc00089a340, {0xc00071bfa8?, 0x594e200f43b0fe1d?, 0x1511f0c0219e05ba?})
	/opt/hostedtoolcache/go/1.19.12/x64/src/testing/testing.go:934 +0x58
github.com/scylladb/scylla-manager/v3/pkg/service/backup_test.TestBackupResumeIntegration.func4.1()
	/home/runner/work/scylla-manager/scylla-manager/pkg/service/backup/service_backup_integration_test.go:1002 +0x2b3
created by github.com/scylladb/scylla-manager/v3/pkg/service/backup_test.TestBackupResumeIntegration.func4
	/home/runner/work/scylla-manager/scylla-manager/pkg/service/backup/service_backup_integration_test.go:997 +0x265
make: *** [Makefile:138: pkg-integration-test] Error 2
```

--- a/pkg/service/backup/service_backup_integration_test.go
+++ b/pkg/service/backup/service_backup_integration_test.go
@@ -218,7 +218,7 @@ func execOnAllHosts(h *CommonTestHelper, cmd string) {
 }
 
 const (
-	maxWaitCond               = 5 * time.Second
+	maxWaitCond               = time.Minute
 	condCheckInterval         = 100 * time.Millisecond
 	longPollingTimeoutSeconds = 2
 )

